bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -354,8 +360,8 @@ impl Script {
         let mut n = 0;
         let mut pushnum_cache = None;
         for inst in self.instructions() {
-            match inst? {
```

Comment:
Why not use question mark? Seemed fine to me.

Reply:
Either way is fine. Reverted.